test/Sentry.Tests/AttributeReaderTests.cs

```diff
@@ -0,0 +1,12 @@
+﻿using AttributeReader = Sentry.AttributeReader;
+
+public class AttributeReaderTests
+{
+    [Fact]
```

Comment:
Cool, this should run on iOS AOT'ed so we would see how it behaves there. Assuming the tests are running there following #1703